exonum-java-binding-common/src/main/java/com/exonum/binding/common/serialization/SerializationUtils.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2018 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.exonum.binding.common.serialization;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+final class SerializationUtils {
+
+  /**
+   * Performs check that serialized value has correct length.
+   *
+   * @param serializedValue serialized value in bytes
+   * @param length expected length
+   * @throws IllegalArgumentException thrown if length is incorrect
+   */
+  static void checkLength(byte[] serializedValue, int length) {
```

Comment:
just `array` — the fact that it is a serialized value does not seem to be reflected anywhere in this code